docs/current_docs/integrations/snippets/gitlab-runner-config.yml

```diff
@@ -0,0 +1,34 @@
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: dagger-custom-config-toml
+data:
+  config.toml: |
+    concurrent = 10
+    [[runners]]
+      environment = ["HOME=/tmp","FF_GITLAB_REGISTRY_HELPER_IMAGE=1", "_EXPERIMENTAL_DAGGER_RUNNER_HOST"="unix:///var/run/dagger/buildkitd.sock"]
+      name = "GitLab Runner with Dagger"
+      url = YOUR-GITLAB-URL
+      executor = "kubernetes"
+      [runners.kubernetes]
+        namespace = "dagger"
+        pull_policy = "always"
+        privileged = true
+      [runners.kubernetes.affinity]
+      [runners.kubernetes.affinity.node_affinity.required_during_scheduling_ignored_during_execution]
+        [[runners.kubernetes.affinity.node_affinity.required_during_scheduling_ignored_during_execution.node_selector_terms]]
+          [[runners.kubernetes.affinity.node_affinity.required_during_scheduling_ignored_during_execution.node_selector_terms.match_expressions]]
+            key = "dagger-node"
+            operator = "In"
+            values = ["true"]
+      [runners.kubernetes.node_tolerations]
+        "dagger-node" = ""
+      [runners.kubernetes.pod_security_context]
+        run_as_non_root = false
+        run_as_user = 0
+        run_as_group = 1001
+        fs_group = 1001
```

Comment:
Double-checking that these are the correct values in GitLab Runner. The expectation is that the system user which runs the `dagger` CLI in the context of the GitLab Runner belongs to the same system group so that it is able to read Dagger's unix socket.

Reply:
@chrira any thoughts on this?

Reply:
Changing those settings did not have any impact on my setup.
I would delete this two lines.

Reply:
Deleted